lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -542,9 +642,19 @@ def test_user_groups(self):
 
     def test_get_redirect_url(self):
         # test the course location
-        assert '/courses/{course_key}/courseware?{activate_block_id}'.format(course_key=str(self.course_key), activate_block_id=urlencode({'activate_block_id': str(self.course.location)})) == get_legacy_courseware_url(self.course.location)  # pylint: disable=line-too-long
```

Comment:
Most of this diff block is breaking up this crazy long line.

The actual change is
-`get_legacy_courseware_url(course.location)`
+`get_courseware_url(course.location, experience=ExperienceOption.LEGACY)`